engine/schema/src/main/resources/META-INF/db/schema-41810to41900.sql

```diff
@@ -468,120 +161,3 @@ ALTER TABLE `cloud`.`snapshot_store_ref`
-         LEFT JOIN `projects` ON ((`projects`.`project_account_id` = `account`.`id`)))
-         LEFT JOIN `volumes` ON ((`volumes`.`id` = `snapshots`.`volume_id`)))
-         LEFT JOIN `snapshot_store_ref` ON (((`snapshot_store_ref`.`snapshot_id` = `snapshots`.`id`)
-             AND (`snapshot_store_ref`.`state` != 'Destroyed')
-             AND (`snapshot_store_ref`.`display` = 1))))
-         LEFT JOIN `image_store` ON ((ISNULL(`image_store`.`removed`)
-             AND (`snapshot_store_ref`.`store_role` = 'Image')
-             AND (`snapshot_store_ref`.`store_id` IS NOT NULL)
-             AND (`image_store`.`id` = `snapshot_store_ref`.`store_id`))))
-         LEFT JOIN `storage_pool` ON ((ISNULL(`storage_pool`.`removed`)
-             AND (`snapshot_store_ref`.`store_role` = 'Primary')
-             AND (`snapshot_store_ref`.`store_id` IS NOT NULL)
-             AND (`storage_pool`.`id` = `snapshot_store_ref`.`store_id`))))
-         LEFT JOIN `snapshot_zone_ref` ON (((`snapshot_zone_ref`.`snapshot_id` = `snapshots`.`id`)
-             AND ISNULL(`snapshot_store_ref`.`store_id`)
-             AND ISNULL(`snapshot_zone_ref`.`removed`))))
-         LEFT JOIN `data_center` ON (((`image_store`.`data_center_id` = `data_center`.`id`)
-             OR (`storage_pool`.`data_center_id` = `data_center`.`id`)
-             OR (`snapshot_zone_ref`.`zone_id` = `data_center`.`id`))))
-         LEFT JOIN `resource_tags` ON ((`resource_tags`.`resource_id` = `snapshots`.`id`)
-             AND (`resource_tags`.`resource_type` = 'Snapshot')));
-
-UPDATE `cloud`.`configuration` SET
-    `options` = concat(`options`, ',OAUTH2'),
-    `default_value` = concat(`default_value`, ',OAUTH2'),
-    `value` = concat(`value`, ',OAUTH2')
-WHERE `name` = 'user.authenticators.order' ;
-
-UPDATE `cloud`.`configuration` SET
-    `options` = concat(`options`, ',OAUTH2Auth'),
-    `default_value` = concat(`default_value`, ',OAUTH2Auth'),
-    `value` = concat(`value`, ',OAUTH2Auth')
-where `name` = 'pluggableApi.authenticators.order' ;
-
--- Create table for OAuth provider details
-DROP TABLE IF EXISTS `cloud`.`oauth_provider`;
-CREATE TABLE `cloud`.`oauth_provider` (
-  `id` bigint unsigned NOT NULL auto_increment COMMENT 'id',
-  `uuid` varchar(40) NOT NULL COMMENT 'unique identifier',
-  `description` varchar(1024) COMMENT 'description of the provider',
-  `provider` varchar(40) NOT NULL COMMENT 'name of the provider',
-  `client_id` varchar(255) NOT NULL COMMENT 'client id which is configured in the provider',
-  `secret_key` varchar(255) NOT NULL COMMENT 'secret key which is configured in the provider',
-  `redirect_uri` varchar(255) NOT NULL COMMENT 'redirect uri which is configured in the provider',
-  `enabled` int(1) NOT NULL DEFAULT 1 COMMENT 'Enabled or disabled',
-  `created` datetime NOT NULL COMMENT 'date created',
-  `removed` datetime COMMENT 'date removed if not null',
-  PRIMARY KEY (`id`)
-  ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
@GutoVeronezi @DaanHoogland @harikrishna-patnala this PR removed some OAuth related changes. Can you please check if that was needed?

Reply:
Perhaps a mistake solving a conflict. I will add it again.

Reply:
Created a draft PR https://github.com/apache/cloudstack/pull/8245/files